src/index.html

```diff
@@ -4,9 +4,315 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="header catalog__header" id="home">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#home" class="nav__link nav__link--is_active is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards-block catalog__block">
+      <div class="card cards-block__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple A1419 iMac"
+          class="card__item-photo"
+        >
+        <h2 class="card__item-title">APPLE A1419 iMac 27" Retina 
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card__item-code">Product code: 195434</h3>
+        <div class="card__reviews">
+          <div class="stars card__stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <div class="card__item-price">
+          <h3 class="card__price-title">Price:</h3>
+          <h3 class="card__price">$2,199</h3>
+        </div>
+        <a
+          href="#"
+          class="card__btn-link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
![2023-02-10_15h03_21](https://user-images.githubusercontent.com/61904995/218098969-59aadcee-65ea-4187-ac2e-84615f4e18dc.png)